Dockerfile

```diff
@@ -1,31 +1,23 @@
 FROM quay.io/ukhomeofficedigital/centos-base:v0.5.0
 
-RUN yum install -y wget && wget http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-8.noarch.rpm && \
-  rpm -ivh epel-release-7-8.noarch.rpm
+RUN yum install -y wget &&  wget http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-9.noarch.rpm 
+RUN rpm -ivh epel-release-7-9.noarch.rpm
 
-RUN yum install -y -q python-pip java-headless fontconfig dejavu-sans-fonts git parallel which; yum clean all; pip install awscli
+RUN yum install -y -q python-pip java-1.8.0-openjdk-devel fontconfig dejavu-sans-fonts git parallel which; yum clean all; pip install awscli
```

Comment:
I think I understand what you are trying to do. I wonder if you could do the same with JDK? Ie, supply a path to the java bin, which is a docker run command?

Either way, this look fine....